components/lib/datatable/BodyRow.js

```diff
@@ -465,10 +465,11 @@ export const BodyRow = React.memo((props) => {
     const content = createContent();
     const tabIndex = getTabIndex();
     const rowProps = mergeProps(
+        getBodyRowPTOptions('bodyRow'),
```

Comment:
Does this really need to be first now?

Reply:
It is needed because otherwise we have bg-primary before bg-white and tailwind merge keep last.

Reply:
Or I can keep in second place and add rowClassName at third place so we minimise the change.